Marlin/language_en.h

```diff
@@ -508,5 +508,16 @@
 #ifndef MSG_DELTA_CALIBRATE_CENTER
   #define MSG_DELTA_CALIBRATE_CENTER          "Calibrate Center"
 #endif
-
+#ifndef MSG_CHANGE_FILAMENT 
+ #define MSG_CHANGE_FILAMENT                 "Filament"
+#endif
+#ifndef MSG_REMOVE_FILAMENT  
+ #define MSG_REMOVE_FILAMENT                 "Remove"
+#endif
+#ifndef MSG_INSERT_FILAMENT 
+ #define MSG_INSERT_FILAMENT                 "Insert"
+#endif
+#ifndef MSG_INSERT_FILAMENT_SLOW 
+ #define MSG_INSERT_FILAMENT_SLOW            "Slow Insert"
+#endif
 #endif // LANGUAGE_EN_H
```

Comment:
:smile: 
also, the travis check has failed at scara configuration :s 
Could u help me?